src/KeyVault/KeyVault/Resources/skr-policy.json

```diff
@@ -0,0 +1,57 @@
+﻿{
```

Comment:
We should make sure that CVM team can reach out to us when the default policy file needs to be updated.

Reply:
emailed to CVM team